jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -184,6 +184,8 @@ def _default_name(branch: pfs.Branch) -> str:
 
 
 class PFSManager(FileContentsManager):
+    mounted_branch: pfs.Branch = None
```

Comment:
This being nullable makes me a bit nervous. With the changes you've made the safe thing to do would be to check `mount_branch is not None` at every call site. I think a compromise approach here is to check the value of mount_branch at all entry points, the "public" methods which are just `download` and `get`, and the still pass `branch` as a parameter within all of "private" (leading underscore) methods. That will at least contain this.

Additionally, I know we are lax with this in this repo, but I'd prefer if this was type hinted as `typing.Optional[pfs.Branch]`.

Reply:
That makes sense will change this.